frontend/src/components/pages/Scheduling/DonationInformation.tsx

```diff
@@ -45,8 +50,20 @@ const DonationInformation: any = ({
       ...formErrors,
       size: "",
     });
+
+    setShowItemTypes(true);
   };
 
+  const checkSubmit = (e: React.ChangeEvent<HTMLInputElement>, item: string) => {
+    const newCategories = e.target.checked ? [...categories, item] : categories.filter((category) => category !== item);
+    
+    if (newCategories.length > 0) {
+      setCanSubmit(true);
+    } else {
+      setCanSubmit(false);
+    }
+  }
```

Comment:
```suggestion
setCanSubmit(!!newCategories.length)
```

Reply:
woah didn't know about this syntax :0 super cool